### ISSUE TITLE
Unable to import the project into Unity

### ISSUE THREAD

**Issue by rinosh1989** [cmdr2/trooper2c#2]
Hi
I downloaded the zip file of the project from repository. I am unable to import into latest version of Unity(5.3.1). I am getting path error.

**Comment by cmdr2** [comment]
Hey, sorry about that. I can reproduce it on my end, working on a fix.

**Comment by rinosh1989** [comment]
Thank you for your prompt reply.
This is the error I got.
![unity](https://cloud.githubusercontent.com/assets/7686819/12173096/e21caa1c-b57a-11e5-9af5-201dfcc31f47.PNG)

**Comment by cmdr2** [comment]
tl;dr: download the new zip, and then 'force quit' once and reopen.

I tried changing a few things, and there is good news and bad news (and bonus news).

Bad news: Try as I may, the error doesn't seem to go away. It now complains about a different file. I'll continue looking into this, but read below:

Good news: I've made a few changes, and now the error only shows up once (for me). That is, if you press 'Force Quit' the first time you see the error, and reopen Unity and the project, it should import successfully. Subsequent opens work fine on my system now. Let me know if that doesn't work for you, and I'll try something else.

Bonus news: I've checked in the long-pending refactor of the code. The previous version was a complete mess and a pain to read and understand. The new version is hopefully a little easier to understand and hack at.

**Comment by rinosh1989** [comment]
Ok. Thanks. I will try and let you know.

**Comment by rinosh1989** [comment]
It worked as you suggested. Thanks for the help.
